pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/creator/name/SimpleSegmentNameGenerator.java

```diff
@@ -40,13 +40,14 @@ public class SimpleSegmentNameGenerator implements SegmentNameGenerator {
   private final String _segmentNamePrefix;
   private final String _segmentNamePostfix;
   private final boolean _appendUUIDToSegmentName;
+  private final boolean _omitTimestampsInSegmentName;
 
   public SimpleSegmentNameGenerator(String segmentNamePrefix, @Nullable String segmentNamePostfix) {
-    this(segmentNamePrefix, segmentNamePostfix, false);
+    this(segmentNamePrefix, segmentNamePostfix, false, false);
```

Comment:
i noticed the use of this constructor for minion here: https://github.com/apache/pinot/blob/master/pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/creator/name/SegmentNameGeneratorFactory.java#L54-L57 but it doesn't seem like `appendUUIDToSegmentName` is threaded through there so figured it wasn't necessary to support this one either.

Reply:
Yes defaults to false, so should be good.